modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/SearchResultsMessage.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {sub} from 'frontend-js-web';
+import React, {useEffect, useState} from 'react';
+
+import isNullOrUndefined from './isNullOrUndefined';
+
+export default function SearchResultsMessage({
+	numberOfResults = null,
+	resultType = Liferay.Language.get('results'),
+}: {
+	numberOfResults: number | null;
+	resultType?: string;
+}) {
+	const [text, setText] = useState('');
+
+	useEffect(() => {
+		if (!isNullOrUndefined(numberOfResults)) {
```

Comment:
Why do you check for undefined? We know that `numberOfResults` is a required prop and has to be either null or a number.

I think we could even remove the `isNullOrUndefined` helper and just do `if (numberOfResults !== null) {`